kernel/packages/shared/profiles/transformations/profileToServerFormat.ts

```diff
@@ -1,20 +1,18 @@
 import { analizeColorPart, stripAlpha } from './analizeColorPart'
 import { isValidBodyShape } from './isValidBodyShape'
-import { Profile } from '../types'
+import { Profile, Snapshots } from '../types'
+import { WearableId } from 'decentraland-ecs/src'
 
-export function ensureServerFormat(profile: Profile) {
+export function ensureServerFormat(profile: Profile): ServerFormatProfile {
   const { avatar } = profile
   const eyes = stripAlpha(analizeColorPart(avatar, 'eyeColor', 'eyes'))
   const hair = stripAlpha(analizeColorPart(avatar, 'hairColor', 'hair'))
   const skin = stripAlpha(analizeColorPart(avatar, 'skin', 'skinColor'))
-  if (
+  const invalidWearables =
     !avatar.wearables ||
     !Array.isArray(avatar.wearables) ||
-    !avatar.wearables.reduce(
-      (prev: boolean, next: any) => prev && typeof next === 'string' && next.startsWith('dcl://'),
-      true
-    )
-  ) {
+    !avatar.wearables.reduce((prev: boolean, next: any) => prev && typeof next === 'string', true)
```

Comment:
We removed the check `next.startsWith('dcl://')` to consider wearables valid, since they won't have that format anymore